packages/query-devtools/src/contexts/QueryDevtoolsContext.ts

```diff
@@ -27,8 +26,10 @@ export interface QueryDevtoolsProps {
   buttonPosition?: DevtoolsButtonPosition
   position?: DevtoolsPosition
   initialIsOpen?: boolean
+  isOpen?: boolean
```

Comment:
@toofff is this prop still required?

Reply:
Sorry, I forgot to delete it. Now I have.